qemu/tests/timedrift_no_net.py

```diff
@@ -21,13 +23,13 @@ def _get_session(self):
     @error_context.context_aware
     def action_during_suspend(self, **args):
         error_context.context("Sleep a while before resuming guest",
-                              logging.info)
+                              LOG_JOB.info)
```

Comment:
`class GuestSuspendBaseTest(utils_test.qemu.GuestSuspend)`, and `utils_test.qemu.GuestSuspend` use `self.test = test`. By definition,  `test` can be anything,   but as ` self.test.fail` be used in derived class `GuestSuspendNegativeTest(GuestSuspendBaseTest)`,   so i think replace it with `self.test.log` is acceptable too.